packages/block-library/src/columns/style.scss

```diff
@@ -99,6 +102,42 @@
 			}
 		}
 	}
+
+	// Divided block style.
+	//
+	// Following styles update the display of borders for this block style. They
+	// require the use of !important to override the inline styles supplied by
+	// the border block support allowing user defined borders.
+	&.is-style-divided .wp-block-column {
+		// Default divided border to use until user specifies something different.
+		border: 1px solid $gray-900;
+
+		// Disable top and left borders.
+		border-top: none !important;
```

Comment:
If at all possible, we should avoid `!important`s. If we need to increase specificity instead, that would be preferable. I could imagine a particular theme wanting to customize this style variation further.

Reply:
> If at all possible, we should avoid !importants. If we need to increase specificity instead, that would be preferable.

Allowing users to configure border colour, style or width via these block support controls means those styles will be applied via inline styles. To overcome those and disable the other sides to give this "Divided" block style `!important` is required.

The comment at the top of that CSS section defining "Divided" styles explains this. Does it need to be made clearer or more prominent?